aten/src/ATen/core/interned_strings.h

```diff
@@ -23,7 +23,6 @@ namespace c10 {
   _(prim, BroadcastSizes)          \
   _(prim, Constant)                \
   _(prim, ChunkSizes)              \
-  _(prim, None)                    \
```

Comment:
Dont think u meant to remove thise

Reply:
shouldn't prim::None already died in your previous PR? I just removed it here to do some clean up, do you think it will break anything?

Reply:
oh yep you're right nvm